Automattic/wp-calypso
# Stats: Update illustration on purchase pages

Related to #86129.

## Proposed Changes

* Adds the new illustration in PNG format.
* Includes support for HiDPI/Retina displays.
* Adds a new wrapper component around the image data.
* Enables the new illustration if the feature flag is set.

Per our research and feedback we decided to go with two PNG files instead of an SVG. This gives us Retina support with a smaller overall hit to our bundle size.

## Purchase page with new illustration
<img width="1169" alt="SCR-20240115-qcxh" src="https://github.com/Automattic/wp-calypso/assets/40267301/b745c279-327e-4003-9823-eabe1e92cf7b">

## Testing Instructions



* Visit the Calypso Live link.
* Visit the Stats page.
* Enable the feature flag: `stats/checkout-flows-v2`
* Confirm the new image is visible and the source is the new PNG data.
* Confirm you see the Retina version of the data (check for `@2x` in the image URL) if using a Retina display.
* Confirm the alt text is visible in the source.

## Pre-merge Checklist
